nautobot/core/api/metadata.py

```diff
@@ -80,13 +83,21 @@ def _get_field_properties(self, field: schema.SerializerType, name: str) -> Dict
             if isinstance(field, drf_serializers.RelatedField):
                 result["uniqueItems"] = True
                 if hasattr(field, "queryset") and hasattr(field.queryset, "model"):
+                    # We construct the correct list view url here so we can just append the object id to the end of the url in our frontend.
+                    # e.g. /dcim/locations/
+                    try:
+                        model_url = reverse(get_route_for_model(model=field.queryset.model, action="list"))
+                    except NoReverseMatch:
+                        model_url = None
```

Comment:
What's the side effect if `model_url` is `None`? It's not immediately clear to me what the outcome would be.

Reply:
The field would not be hyperlinked. ContentType fields, for example, do not have a detail view in Nautobot.